pkg/profiler/cpu/cpu.go

```diff
@@ -673,7 +767,7 @@ func (p *CPU) Run(ctx context.Context) error {
 				pi.Mappings.ExecutableSections(),
 				p.LastProfileStartedAt(),
 				samplingPeriod,
-			).Convert(ctx, perProcessRawData.RawSamples)
+			).Convert(ctx, perProcessRawData.RawSamples, p.bpfMaps.InterpreterFrames)
```

Comment:
🔧 I believe `p.bpfMaps.InterpreterFrames` should be passed to the `NewConverter` factor method. It's used as a lookup dependency, so it better fits there as a dependency.